Source/Venice/Channel/Channel.swift

```diff
@@ -67,26 +67,23 @@ public final class Channel<T>: Sequence {
     }
 
     /// Closes the channel. When a channel is closed it cannot receive values anymore.
-    public func close() -> Bool {
-        if closed {
-            return false
-        }
+    public func close() throws {
+        guard !closed else { return }
```

Comment:
Can `BinaryStream` inherit from `Stream` and "concretize" the `associatedType`? is that possible?